documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -107,9 +121,97 @@ EOF
 <2> Specify the _managed object reference_ of the VMware host.
 <3> Specify the IP address of the VMware migration network.
 
+. If you want to map multiple networks, create a `NetworkAttachmentDefinition` CR manifest for each additional destination network:
++
+[source,yaml]
+----
+$ cat << EOF | oc create -f -
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: <networkattachmentdefinition_name>
+spec:
+  config: '{
+    "cniVersion": "0.4.0",
+    "name": "<networkattachmentdefinition_name>",
+    "plugins": [
+      {
+        "type": "cnv-bridge",
+        "bridge": "br1",
+        "vlan": 1 <1>
+      },
+      {
+        "type": "cnv-tuning" <2>
+      }
+    ]
+  }'
+EOF
+----
+<1> Optional.
+<2> The `cnv-tuning` plug-in allows the MAC pool manager to assign a unique MAC address to the connection.
+
+. Create a `NetworkMap` CR manifest to map the source and destination networks:
++
+[source,yaml,subs="attributes+"]
+----
+$  cat << EOF | oc apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: NetworkMap
+metadata:
+  name: <networkmap_name>
+  namespace: {namespace}
+spec:
+  map:
+    - destination:
+        type: pod <1>
+      source:
+        id: <source_network_mor> <2>
+    - destination:
+        type: multus <1>
+        name: <networkattachmentdefinition_name>
+      source:
+        id: <source_network_mor> <2>
+  provider:
+    source:
+      name: vmware-provider
+      namespace: {namespace}
+    destination:
+      name: destination-cluster
+      namespace: {namespace}
+EOF
+----
+<1> Allowed values are `pod` and `multus`.
+<2> Specify the managed object reference of the source network.
+
+. Create a `StorageMap` CR manifest:
++
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | oc apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: StorageMap
+metadata:
+  name: <storagemap_name>
+  namespace: {namespace}
+spec:
+  map:
+    - destination:
+        storageClass: <destination_storageclass>
+      source:
+        id: <source_datastore_id>
```

Comment:
id: <source_datastore_mor>

I would like to know what "mor" stands for, but I saw you used it in other places